prometheus_flask_exporter/__init__.py

```diff
@@ -146,6 +149,7 @@ def __init__(self, app, path='/metrics',
         self._default_latency_as_histogram = default_latency_as_histogram
         self._response_converter = response_converter or make_response
         self._metrics_decorator = metrics_decorator
+        self._export_defaults_with_labels = export_defaults_with_labels or True
```

Comment:
If we do keep this PR, `export_defaults_with_labels` is `None` makes this `True`, but `export_defaults_with_labels` is `False` also makes this `True`.
In other words, this can never be NOT `True`.

Reply:
sorry just reply this. yes you are right regarding `It sounds like maybe you don't want those to be default labels, but just add the same labels to your custom metrics maybe?`.

what I meant, if user don't use `export_defaults_with_labels` param , it will be True which means they want their custom labels to be included as well to default metrics. thus, if they want to exclude their custom labels to be included to default metrics, they need to specifically mention the `export_defaults_with_labels` and make it `False` thus it can be True because the `export_defaults_with_labels` param exist

Reply:
No worries.

OK, I think I see your point. My workaround above should give this to you, but there's an argument for having default labels you only want to add to your custom metrics I assume.

I'm happy to give that a go, or you feel like getting this PR in, can you please add tests to make sure this new parameter is covered by them?

Reply:
ok will do, got to steal a free time for this :) , but will be glad to, thanks for your response